src/discriminant-calculator.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+int discriminantCalculator(int a, int b, int c) {
+	int res = pow(b, 2) - 4 * a * c;
+	int x1 = (-b + sqrt(res)) / (2 * a);
+	int x2 = (-b - sqrt(res)) / (2 * a);
+	
+	printf("Discriminant = %d \n", res);
+	printf("x1 = %d\nx2 = %d \n", x1, x2);
+
+	if (res < 0) {
+		printf("Discriminant < 0, has 2 complex roots");
+	}
+
+	return 1;
+}
+
+int main(int argc, char* argv[]) {
+	int a = atoi(argv[1]);
+	int b = atoi(argv[2]);
+	int c = atoi(argv[3]);
+
+	if (argc < 4) {
+		printf("Not enough values, enter 3 numbers");
+	} else if (argc > 4) {
+		printf("Too many values, enter 3 numbers");
+	} else {
+		discriminantCalculator(a, b, c);	
```

Comment:
Implement the function for parsing the result of the `discriminantCalculator`.
It will receive the result and based on the result do print (for example)